addons/web_editor/static/src/js/wysiwyg/widgets/media.js

```diff
@@ -867,10 +867,10 @@ var IconWidget = SearchableMediaWidget.extend({
             var cls = classes[i];
             if (_.contains(this.alias, cls)) {
                 this.selectedIcon = cls;
+                this.initialIcon = cls;
                 this._highlightSelectedIcon();
             }
         }
-        this.nonIconClasses = _.without(classes, 'media_iframe_video', this.selectedIcon);
```

Comment:
Not to be removed from `this` in a stable version.

**If you must**, you can stop using it to fix an issue (so potentially making a custo using it not *working* anymore as could be relying on the Odoo code using it too) but you **cannot** remove it entirely as custo using it would simply crash on the line using it for no reason. So: comment the line to be removed in master but do not remove it.

Reply:
Since the place where the "non icon classes" are copied over to the new element is now no longer part of the widget lifecycle (handled in `save().then(...)`, I don't see a good way I can keep using it. I'll add it back with a comment to remove in master